submissions/TaniaBondarenko/friends-app/app.js

```diff
@@ -0,0 +1,130 @@
+const numOfFriends=20;
+const apiUrl = `https://randomuser.me/api/?results=${numOfFriends}`;
+const friends = document.querySelector(".friends");
+let friendsData;
+let arrForFilter;
+let isFiltered = false;
+let justMale;
+let justFemale;
+
+function makeContainerEmpty() {
+        let innerText = " "; 
+        friends.innerHTML = innerText.replace(innerText, " ");    
+}
+
+function fetchFriend() {
+        fetch(apiUrl)
+                .then(response => response.json())
+                .then((data) => friendsData = data.results)
+                .then(() => addFriends(friendsData))
+                .catch(err => {
+                        refreshPage();
```

Comment:
Why do you need to refresh the page in case of an error?

Reply:
I thought it might be user friendly action.

Reply:
Well... Not exactly. Imagine, if FB will reload in case of error. Will it be friendlier? No.

If you have an error you can show error to the user or re-fetch data

Reply:
Okay